src/vstart.sh

```diff
@@ -1412,3 +1412,16 @@ echo ""
 
 echo "CEPH_DEV=1"
 
+# always keep this section at the very bottom of this file
+STRAY_CONF_PATH="/etc/ceph/ceph.conf"
+if [ -f "$STRAY_CONF_PATH" ] && [ -n "$conf_fn" ] && [ "$conf_fn" != "$STRAY_CONF_PATH" ]; then
```

Comment:
Third test should be `-ef` instead of `!=`. Also collapse into single test:

`if [ -f ... -a -n ... -a ...]; then`

Reply:
done ... corrected the condition check